Add dependency in composer.json

Schrank opened the pull request:
Without  "magento-hackathon/magento-composer-installer": "^3.2" module is not installed

Schrank committed:
Merge pull request #5 from mage-one-com/develop

T28444 : Test für QPS bauen

Schrank committed:
Merge pull request #11 from mage-one-com/develop

Improve documentation/informations

riconeitzel committed:
Merge pull request #13 from mage-one-com/develop

FIX ACL for rule controller

Schrank committed:
Add dependency in composer.json

Without  "magento-hackathon/magento-composer-installer": "^3.2" module is not installed

Schrank committed:
Update composer.json

Add missing "

riconeitzel commented:
@aadmathijssen does it work for you now? :-)

aadmathijssen commented:
Hi @Schrank and @riconeitzel,

The error message is gone now, thanks for the quick fix.

Regarding the "require" vs "suggest" discussion, the Magento Composer Installer FAQ [states the following](https://github.com/Cotya/magento-composer-installer/blob/main/doc/FAQ.md#should-my-module-require-the-installer):

> **Should my module require the Installer**
> 
> No, it should not. But it can suggest using it (same syntax as require, but you use "suggest" instead of "require".
> 
> Why? Because there can be alternatives to this Installer or some people dont want to use an installer for them at all. As a module publisher you only decide the type of your package. It is the responsibility of the package user to decide what installer or plugin he uses to handle this package type. Also by require the installer, you can produce version conflicts, as people could start require specific versions of the Installer. Which makes absolutely no sense for a module.

Schrank commented:
Convinced! 😂

Schrank committed:
Update composer.json

Do not require but suggest - there might be other installed who do the same job.
